packages/storybook/cxl-ui/cxl-vaadin-accordion/cxl-playbook-accordion.story.js

```diff
@@ -2,19 +2,121 @@ import { html } from 'lit-html';
 import { unsafeHTML } from 'lit-html/directives/unsafe-html';
 import playbookStepData from './cxl-playbook-accordion.data.json';
 
+// let tmpTimeoutId = 0;
+//
+// const hideAll = () => {
+//   document.querySelectorAll(`[icon="vaadin:comment"]`).forEach((el) => {
+//     el.classList.add('hidden');
+//   });
+// };
+//
+// const clearTimeout = () => {
+//   window.clearTimeout(tmpTimeoutId);
+// };
+//
+// const setDelay = () => {
+//   clearTimeout();
+//   tmpTimeoutId = window.setTimeout(() => hideAll(), 2000);
+// };
+//
+// const isLayoutWide = () => document.querySelector('cxl-app-layout').wide;
+//
+// const onload = () => {
+//   window.addEventListener('DOMContentLoaded', () => {
+//     document.querySelectorAll('cxl-playbook-accordion').forEach((el) => {
+//       el.addEventListener('mouseleave', () => {
+//         setDelay();
+//       });
+//     });
+//
+//     document.querySelectorAll('vaadin-accordion-panel').forEach((el) => {
+//       el.addEventListener('mouseenter', (evt) => {
+//         if (!isLayoutWide()) {
+//           return;
+//         }
+//         setDelay();
+//
+//         const stepId = evt.target.getAttribute('data-step-id');
+//         const commentIcon = document.querySelector(
+//           `header [icon="vaadin:comment"][data-step-id="${stepId}"]`
+//         );
+//
+//         hideAll();
+//
+//         if (commentIcon) {
+//           commentIcon.classList.remove('hidden');
+//         }
+//       });
+//
+//       el.addEventListener('mouseleave', () => {
+//         if (!isLayoutWide()) {
+//           return;
+//         }
+//         setDelay();
+//       });
+//
+//       el.querySelectorAll('iron-icon').forEach((icon) => {
+//         icon.addEventListener('mouseenter', () => {
+//           clearTimeout();
+//         });
+//
+//         icon.addEventListener('mouseleave', () => {
+//           setDelay();
+//         });
+//
+//         icon.addEventListener('click', (evt) => {
+//           // here should go the comment popup handler click
+//           evt.stopImmediatePropagation();
+//         });
+//       });
+//     });
+//   });
+// };
+
```

Comment:
We'll go with nested button for now, so this JS can be dropped. It will continue to live in this history for a while, if we ever need to bring it back.

For future - if code starts looking weirdly micro-managing things similar to the above, I recommend: stop and get consultation for alternative approaches quickly.

Reply:
@lkraav I didn't know we actually have any alternatives to the icon. Remove this and leave it only in the `git` history, or leave it commented out to quickly turn it on in case we need it ?

Reply:
Remove.

Reply:
Never mind, I just dropped the icon in one commit along with the commented JS.